site/app/templates/notebook/Notebook.twig

```diff
@@ -130,6 +131,7 @@
 
                 <fieldset id="mc_field_{{ num_multiple_choice }}"
                           class="mc"
+                          name="{{ cell.filename }}"
```

Comment:
Same suggestion as above, can change to `data-filename` instead of name attribute